shell/platform/linux/fl_engine_private.h

```diff
@@ -0,0 +1,93 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_ENGINE_PRIVATE_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_ENGINE_PRIVATE_H_
+
+#include <glib-object.h>
+
+#include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/linux/fl_renderer.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_dart_project.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_engine.h"
+
+G_BEGIN_DECLS
+
+/**
+ * fl_engine_new:
+ * @project: a #FlDartProject
+ * @renderer: a #FlRenderer
+ *
+ * Creates a new Flutter engine.
+ *
+ * Returns: a #FlEngine
+ */
+FlEngine* fl_engine_new(FlDartProject* project, FlRenderer* renderer);
+
+/**
+ * fl_engine_start:
+ * @engine: a #FlEngine
+ * @error: (allow-none): #GError location to store the error occurring, or %NULL to ignore.
+ *
+ * Starts the Flutter engine.
+ *
+ * Returns: %TRUE on success
+ */
+gboolean fl_engine_start(FlEngine* engine, GError** error);
+
+/**
+ * fl_engine_get_project:
+ * @engine: a #FlEngine
+ *
+ * Gets the project being run by this engine.
+ *
+ * Returns: a #FlDartProject
+ */
+FlDartProject* fl_engine_get_project(FlEngine* engine);
```

Comment:
Remove, per previous review comment.

Reply:
This is private API (previously it was public), did you still want it removed?

Reply:
We could also remove fl_engine_get_renderer under the same logic.

Reply:
Yes; you're not using it, and I know that at least iOS and macOS have never needed an equivalent method. I'm very strongly in favor of not adding things we don't have any clear expectation of needing. It's easy to add things later, whereas unused cruft methods often stick around and add pointless code (which needs to be maintained during refactors, tested, etc.)

Reply:
OK, I'll kill all unused methods.